### PR TITLE
Correct links to Starburst

### PR DESCRIPTION
**Pull request by mosabua**
## What are you changing in this pull request and why?

See title

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

### CODE REVIEW

File: website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md [hunk 1]
```diff
@@ -398,12 +398,12 @@ Later, you can connect your business intelligence (BI) tools to these views and
...
 
 ## Related content
 - Learn more with the [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) course for free
 - [dbt Courses](https://courses.getdbt.com/collections)
 - [dbt Cloud CI job](/docs/deploy/cloud-ci-job)
 - [Job notifications](/docs/deploy/job-notifications)
 - [Source freshness](/docs/deploy/source-freshness)
-- [SQL basics for Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/tutorials/learn-basics.html)
\ No newline at end of file
+- [SQL overview for Starburst Galaxy](https://docs.starburst.io/starburst-galaxy/sql/index.html)
```
Comment (mosabua) [inline website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md:409]: The overview contains a link to the tutorial and much more other useful links and info

File: website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ id: "starburst-galaxy"
 sidebar_label: "Starburst Galaxy quickstart"
 ---
 
-In this quickstart guide, you'll learn how to use dbt Cloud with [Starburst Galaxy](https://www.starburst.io/). It will show you how to:
+In this quickstart guide, you'll learn how to use dbt Cloud with [Starburst Galaxy](https://www.starburst.io/platform/starburst-galaxy/). It will show you how to:
 
-- Load data into the Amazon S3 bucket. This guide uses AWS as the cloud service provider for demonstrative purposes. Starburst Galaxy also [supports other providers](https://www.starburst.io/platform/connectors/) such as Google Cloud, Microsoft Azure, and more. 
+- Load data into the Amazon S3 bucket. This guide uses AWS as the cloud service provider for demonstrative purposes. Starburst Galaxy also [supports other data sources](https://docs.starburst.io/starburst-galaxy/catalogs/index.html) such as Google Cloud, Microsoft Azure, and more.
```
Comment (mosabua) [inline website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md:10]: The old link is NOT applicable to Starburst Galaxy

**Review by nghi-ly — APPROVED** [review]
great catches, @mosabua, ty ! 🚢

**Comment by nghi-ly** [comment]
went ahead and merged for you, @mosabua . hope that's ok!

**Comment by mosabua** [comment]
Thank you for the merges. I don't think I have access to merge myself anyway.

**Pull request merged**